hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/schema/CompactSchemaReplicationSlowTest.java

```diff
@@ -85,8 +101,6 @@ public void testSchemaReplication_whenAMemberThrowsRetryableExceptionAllTheTime_
     @Override
     public Config getConfig() {
         Config config = super.getConfig();
-        // Jet prints too many logs during the test
```

Comment:
These lines are not related with the problem. It seems implicitly `jet` is re-enabled. But it is ok, it will not affect the test.